tests/stubtest_allowlists/py3_common.txt

```diff
@@ -491,6 +493,20 @@ distutils.unixccompiler.UnixCCompiler.xcode_stub_lib_extension
 distutils.unixccompiler.UnixCCompiler.xcode_stub_lib_format
 distutils.util.grok_environment_error
 
+# ==========
+# Missing from pkg.ImpLoader (deprecated since 3.3, can add if someone needs it)
+# ==========
```

Comment:
- This comment is quite large, considering there's not many lines after it
- Typo: `pkg` --> `pkgutil`

```suggestion
# Missing from pkgutil.ImpLoader (deprecated since 3.3, can add if someone needs it)
```